specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2019-03-01/SmartDetectorAlertRulesApi.json

```diff
@@ -214,6 +214,55 @@
           }
         }
       },
+      "patch": {
+        "tags": [
+          "SmartDetectorAlertRules"
+        ],
+        "operationId": "SmartDetectorAlertRules_Patch",
+        "description": "Patch a specific Smart Detector alert rule.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AlertRuleNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "description": "Parameters supplied to the operation.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AlertRulePatchObject"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/SmartDetectorErrorResponse"
```

Comment:
This error response is different from ARM's guideline. so that is why cause Linter error R4007,
But if your service already implement error response like what your define in SmartDetectorErrorResponse. then we can also suppression

Reply:
We need to keep the error response consistent , can you please suppress this error?  @akning-ms